appveyor.yml

```diff
@@ -4,8 +4,8 @@ pull_requests:
 image: WMF 5
 init:
 - ps: Install-PackageProvider NuGet -Force
-- ps: (new-object net.webclient).DownloadFile('https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.exe', "c:/dotnet-install.exe")
-- cmd: c:\dotnet-install.exe /install /quiet
+- ps: (new-object net.webclient).DownloadFile('https://raw.githubusercontent.com/dotnet/cli/master/scripts/obtain/dotnet-install.ps1', "C:/dotnet-install.ps1")
+- ps: C:/dotnet-install.ps1
```

Comment:
This is working in our project, but we have this code as well:
https://github.com/PowerShell/PowerShell/blob/master/build.psm1#L2088